### ISSUE TITLE
Messages should be opaque

### ISSUE THREAD

**Issue by mkhraisha** [w3c-ccg/traceability-interop#483]
Our tests currently test for the content of the error message to match a specific error schema, for example we check that any 401 response contains a `message` field with the following string:
```
Unauthorized: This endpoint requires an OAuth2 bearer token
```
we should instead be testing that we are responding with the correct status code `401` and that a `message` field is set, without caring about the exact string of the message.

**Comment by brownoxford** [comment]
@mkhraisha are you suggesting that we keep the existing schema properties and simply remove the specific values for `message`, i.e., use `schemas/Error.yml` for all errors instead of overriding with a specific value for `message` as in `responses/BadRequest.yml`?

**Comment by brownoxford** [comment]
Discussed on call. One of the reasons that we have required messages is for tests to be able to verify that failure occurred for the right reason (differentiating between different reasons for a 400, for example). If we don't really care about that, we can remove the message.

**Comment by nissimsan** [comment]
Skipping for next meeting, so @mkhraisha to participate in the discussion.

**Comment by nissimsan** [comment]
Skipping for next meeting, so @mkhraisha to participate in the discussion.

**Comment by OR13** [comment]
This issue is complete when tests assert the presence of a message, and not the specific string values.

**Comment by nissimsan** [comment]
@mkhraisha , please link your PR which fixes this.

**Comment by mkhraisha** [comment]
@nissimsan  good to close as it is  fixed in #569
